spec/features/admins/merchants/index_spec.rb

```diff
@@ -20,10 +24,89 @@
       expect(page).to have_link(merchant_1.name)
       expect(page).to have_link(merchant_2.name)
       expect(page).to have_link(merchant_3.name)
+      expect(page).to have_link(merchant_4.name)
+      expect(page).to have_link(merchant_5.name)
 
       click_link(merchant_1.name)
 
       expect(current_path).to eq(admin_merchant_path(merchant_1))
+
+      visit admin_merchants_path
+
+      click_link(merchant_2.name)
+
+      expect(current_path).to eq(admin_merchant_path(merchant_2))
     end
+    
+    describe 'enable and disable' do
```

Comment:
In the future, you can dry this kind of repetitive test up a bit by storing all the merchant objects and iterating through them. You can have expectations within each blocks and it'll continue to test them all the way to the end.